lib/application/constants/strings.dart

```diff
@@ -0,0 +1,16 @@
+// This is a temporary constants file related to all strings
+//
+// TODO(matuella): update this approach to one that takes the localization (BuildContext)
```

Comment:
But that's not a issue, as we may never update the Localization, thus this won't need to never use `BuildContext`

Reply:
But if this feature may never be implemented, what is the purpose of this TODO? I mean, or this is an enhancement issue or this TODO shouldn't be here because if someday we decide to implement it, it should be handled as a new feature.

Otherwise this TODO will stay here indefinitely, because as you said, this may never be updated